test/discrete_test.json

```diff
@@ -1376,6 +1376,33 @@
     }
   ], 
   [
+    "Poisson(0.0)", 
```

Comment:
Okay, since it tests values against scipy, which does not behave correctly with Poisson(0.0), I've had to fix predref.py as well as adding it to discrete_test.lst. I've added a new pull request as I don't seem to be able to reopen this.